### ISSUE TITLE
GL tracker entries should be linked to the Project Charters

### ISSUE THREAD

**Issue by Redmayspecial** [unfoldingWord-dev/translationDatabaseWeb#446]
The information from the GL tracker should lead to more information about the language.

![image](https://cloud.githubusercontent.com/assets/11600863/13325721/06c36f20-dbb1-11e5-9b38-85749adf6639.png)

Have the name link to the language page (e.g) http://td.unfoldingword.org/uw/languages/356/

![image](https://cloud.githubusercontent.com/assets/11600863/13325735/1b38e124-dbb1-11e5-92e6-e577b8df9f37.png)

**Comment by rbnswartz** [comment]
This is near completion but there are some usability issues I would like to sort out as well as somethings that I'm not quite 100% satisfied with yet.

**Comment by rbnswartz** [comment]
This is taken care of in #499

**Comment by rbnswartz** [comment]
Waiting on merge

**Comment by rbnswartz** [comment]
Merged into production
